frontend/src/components/ApartmentCard/ApartmentCard.tsx

```diff
@@ -30,34 +37,44 @@ const ApartmentCard = ({ buildingData, numReviews, company }: Props): ReactEleme
 
   return (
     <Card className={classes.card}>
-      <CardMedia className={classes.img} image={img} component="img" title={name} />
-      <CardContent>
-        <Grid container spacing={1}>
-          <Grid container item justify="space-between" alignItems="center">
-            <Grid item>
-              <Grid container alignItems="center">
-                <Typography variant="h6" className={classes.nameText}>
+      <Grid container direction="row">
+        <Grid item md={2}>
+          <CardMedia className={classes.img} image={img} component="img" title={name} />
```

Comment:
Try experimenting with `borderRadius` directly on the image - right now it shows up on the `Card` container but sometimes not on the image itself after you add padding so that there's space between the image and the containing `Card` component (add padding to the `<Grid item>` component containing `CardMedia`